src/Lorentz/Contracts/BaseDAO/Types.hs

```diff
@@ -143,9 +145,9 @@ defaultConfig = Config
       , FA2.tmExtras = mempty
       }
   , cProposalCheck = do
-      drop; push True
+      drop; drop; push True
```

Comment:
Btw, there should be `dropN @2` and it's likely more efficient.

Reply:
Updated this.